src/components/ex-article/ex-article.scss

```diff
@@ -0,0 +1,140 @@
+.cmp-article {
+  @apply flex flex-col;
+
+  &__elements {
+    @screen md {
+      @apply p-0 m-0;
+    }
+
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      @apply text-primary pt-4 font-bold leading-8;
```

Comment:
As general remark for this and all future websites: The headline must visually always be closer to the following paragraph than the preceding paragraph.

This is just an example without definitive pixel values. It illustrate the idea:
![image](https://user-images.githubusercontent.com/88790097/133630926-7e5d9cfa-cbfc-43b3-a4f8-52937b224f94.png)


![image](https://user-images.githubusercontent.com/88790097/133630617-954bd95b-a659-4230-b3cb-039d26e4c526.png)

![image](https://user-images.githubusercontent.com/88790097/133630691-b5ee6ef0-2b36-4896-9dae-b68b0d0fc1f7.png)

Reply:
Changed spacing also adding link highlighting in text